superset-frontend/packages/superset-ui-chart-controls/src/shared-controls/customControls.tsx

```diff
@@ -54,27 +54,29 @@ export const contributionModeControl = {
   },
 };
 
+function isNotTemporal(controls: ControlStateMapping): boolean {
```

Comment:
Nit, feel free to ignore if you disagree - I think it's more generic/easier to understand when function's name is positive, not negative, i.e. I'd go for `isTemporal` and then use it with `!` operator

Reply:
Makes total sense, updated 👍